test/pages/WaypointsSpec.scala

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package pages
+
+import org.scalatest.{EitherValues, OptionValues}
+import org.scalatest.freespec.AnyFreeSpec
+import org.scalatest.matchers.must.Matchers
+
+class WaypointsSpec extends AnyFreeSpec with Matchers with OptionValues with EitherValues {
+
+  "EmptyWaypoints" - {
+
+    ".setNextWaypoint" - {
+
+      "must return Waypoints with this waypoint at the head" in new Fixture {
+
+//        EmptyWaypoints.setNextWaypoint(testWaypoint1)
```

Comment:
why is this commented out?

Reply:
Index view removed in commit 1972a30 yesterday. Commented out code can be removed, left there as a placeholder.